sdcm/tester.py

```diff
@@ -1687,6 +1689,9 @@ def get_cluster_k8s_eks(self, n_k8s_clusters: int):  # pylint: disable=too-many-
         ParallelObject(timeout=7200, num_workers=n_k8s_clusters, objects=self.k8s_clusters).run(
             func=eks.deploy_k8s_eks_cluster, unpack_objects=True, ignore_exceptions=False)
 
+        assert isinstance(
+            self.params, SCTConfiguration), "sdcm.tester.ClusterTester.get_cluster_k8s_eks, 2, params type dict"
+
```

Comment:
Why line 1701 is not changed to the `params=init_and_verify_sct_config(),`?

Reply:
I found that `deepcopy` do not change type of parameter. It remain `SCTConfiguration`